src/routes/Dashboard/SummaryCard/components/Fraction.jsx

```diff
@@ -0,0 +1,14 @@
+import PropTypes from "prop-types";
+
+export default function CounterSection({value, label}) {
+    if (!Math.ceil(value)) return null
```

Comment:
store the `Math.ceil` result in a constant to reuse everywhere, you're repeating `Math.ceil` in so many parts